conf/application.conf

```diff
@@ -362,3 +363,6 @@ health {
 # performed. The IP address is looked for in the HTTP header of requests
 # 'true client IP'
 http-header-ip-whitelist = [ ]
+
+# to learn why this was included: https://confluence.tools.tax.service.gov.uk/display/TEC/2016/03/14/Setting+Security+Headers+in+frontend+services
+play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline' localhost:9032 localhost:9250 www.google-analytics.com data:"
```

Comment:
i think we already had this setting in `application.conf` - this looks like a duplicate